File: src/dataset/javascript/bob_exercism-ThomasZumsteg/claude/ClaudeSonnet4_bob_v1.js
var Bob = function() {};

Bob.prototype.hey = function(input) {
	var trimmed = input.trim();
	
	if (!trimmed) {
		return "Fine. Be that way!";
	}
	
	var hasLetters = /[A-Za-z]/.test(trimmed);
	var isAllUpperCase = hasLetters && trimmed === trimmed.toUpperCase();
	var isQuestion = trimmed.charCodeAt(trimmed.length - 1) === 63;
	
	if (isAllUpperCase) {
		return "Whoa, chill out!";
	}
	
	if (isQuestion) {
		return "Sure.";
	}
	
	return "Whatever.";
};

module.exports = Bob;

File: src/dataset/python/bob_exercism-ThomasZumsteg/claude/ClaudeSonnet4_bob_v1.py
"""Replys to statements as a lackadaisical teenager would"""

def hey(what):
    """Replys to statements as a lackadaisical teenager would"""
    what = what.strip()
    if not what:
        return 'Fine. Be that way!'
    elif what.isupper():
        return 'Whoa, chill out!'
    elif what.endswith('?'):
        return 'Sure.'
    else:
        return 'Whatever.'